target/linux/ramips/image/mt7621.mk

```diff
@@ -359,6 +359,15 @@ define Device/sk-wb8
 endef
 TARGET_DEVICES += sk-wb8
 
+define Device/strong_wr1201-zc01-01
+  DTS := WR1201-ZC01-01
+	IMAGE_SIZE := $(ralink_default_fw_size_16M)
+  DEVICE_TITLE := Strong Gigabit Dual Band 1200
+	UIMAGE_NAME:= WR1201_8_128
+  DEVICE_PACKAGES := kmod-sdhci-mt7620 kmod-mt76 kmod-usb3 kmod-usb-ledtrig-usbport wpad-mini
```

Comment:
indentations are mixed here - should be two spaces only

Reply:
> indentations are mixed here - should be two spaces only

Fixed!